backend-db/plugins/accounts.js

```diff
@@ -13,6 +13,7 @@ const filterBriefAccountData = account => util.object.filter(account, {
     role: true,
     locked: true,
     country: true,
+    team: true,
```

Comment:
If this is returned as part of the account properties it SHOULD also be described in the API yaml.